CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,4166 @@
+	}
+	
+	option = {
+    	name = "There is no need to interfere"
+		relation = { who = CRO value = -50 }
+		diplomatic_influence = { who = CRO value = -50 }
+		ai_chance = { factor = 0.05 }
+	}
+
+}
+
+#Muggia agreement - 1
+country_event = {
+	id = 8020439
+	title = "The Muggia Agreement activation"
+	desc = EVT_8020439_DESC
+	picture = "nwo2_free_city_of_trieste"
+  
+	fire_only_once = yes
+
+	trigger = {
+		tag = CRO
+		year = 1992
+		NOT = { has_global_flag = 1992_start_date_flag }
+		
+		government = populist_dictatorship
+		is_vassal = no
+		war = no
+		
+		#ITA exists
+		ITA = {
+			exists = yes
+			is_vassal = no
+			war = no
+		}
+		
+		ITA = { has_country_flag = ITA_CRO_Koper } 
+		has_global_flag = ITA_CRO_Ustasha_funding
+		has_global_flag = ITA_Operation_Tora_Tora 
+	}
+
+	mean_time_to_happen = { months = 1 }
+	
+	immediate = { clr_global_flag = ITA_CRO_Ustasha_funding ITA = { clr_country_flag = ITA_CRO_Koper } }
+
+	option = {
+		name = "Stick to the pacts"
+		prestige = 10
+		
+		random_country = {
+			limit = { tag = ITA is_vassal = no }
+			relation = { who = THIS value = 100 }
+		}
+		
+		SLO = { 3345 = { secede_province = ITA change_province_name = "Capodistria" } } #Koper, Istria
+		ai_chance = { factor = 0.9 }
+	}
+
+	option = {
+		name = "Betray Italy"
+		badboy = 3 
+		
+		prestige = -50
+		
+		random_country = {
+			limit = { tag = ITA is_vassal = no }
+			relation = { who = THIS value = -150 }
+			add_casus_belli = {
+	            target = CRO
+	            type = humiliate
+	            months = 24
+            }
+		}
+		
+		ai_chance = { factor = 0.1 }
+	}
+	
+}
+
+#Muggia agreement - 2
+country_event = {
+	id = 8020440
+	title = "The Muggia Agreement activation"
+	desc = EVT_8020440_DESC
+	picture = "CRO_Greater_Croatia"
+  
+	fire_only_once = yes
+
+	trigger = {
+		tag = CRO
+		year = 1992
+		NOT = { has_global_flag = 1992_start_date_flag }
+		
+		government = populist_dictatorship
+		is_vassal = no
+		war = no
+		
+		#ITA exists
+		ITA = {
+			exists = yes
+			is_vassal = no
+			war = no
+		}
+		
+		ITA = { has_country_flag = ITA_CRO_Istria } 
+		has_global_flag = ITA_CRO_Ustasha_funding
+		has_global_flag = ITA_Operation_Tora_Tora 
+	}
+
+	mean_time_to_happen = { months = 1 }
+	
+	immediate = { clr_global_flag = ITA_CRO_Ustasha_funding ITA = { clr_country_flag = ITA_CRO_Istria } }
+
+	option = {
+		name = "Stick to the pacts"
+		prestige = 10
+		
+		create_alliance = ITA
+		
+		random_country = {
+			limit = { tag = ITA is_vassal = no }
+			relation = { who = THIS value = 150 }
+			add_casus_belli = {
+	            target = BOS
+	            type = free_allied_cores
+	            months = 360
+            }
+			add_casus_belli = {
+	            target = MON
+	            type = free_allied_cores
+	            months = 360
+            }
+			add_casus_belli = {
+	            target = SER
+	            type = free_allied_cores
+	            months = 360
+            }
+		}
+		
+		#Italy
```

Comment:
Territory cannot be taken away unilaterally from an independent nation, there must be an event for the nations below to either accept or reject. Also, there must be a check for SLO and CRO owning the provinces to be transferred.